src/test/kotlin/ecommerce/ApplicationTests.kt

```diff
@@ -0,0 +1,13 @@
+package ecommerce
+
+import org.junit.jupiter.api.Test
+import org.springframework.boot.test.context.SpringBootTest
+import org.springframework.test.context.ActiveProfiles
+
+@SpringBootTest
+@ActiveProfiles("test")
+class ApplicationTests {
+    @Test
+    fun contextLoads() {
+    }
+}
```

Comment:
I don't think this test is necessary in our current state of this project.
Why don't we remove it?